src/site-config.ts

```diff
@@ -12,6 +12,7 @@ type SiteConfig = {
     name: string;
     url: string;
   }[];
+  enableSearch: boolean;
```

Comment:
Hmm, I was thinking that maybe we use environment variables for this. That way we can conditionally also handle the build step.

Want to give it a try?

Reply:
Sure, I think enableSearch: import.meta.env.ENABLE_SEARCH should do the trick.